### PR TITLE
Add Swagger

### PR DESCRIPTION
**Pull request by yasersharifi**
Adding swagger document for api

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 // express app
 const app = express();
 const router = require("./src/routes");
 
+
 require("./start/config")(app, express);
 require("./start/db")(app);
 require("./start/logging")();
 
+// swagger
```
Comment (amirmalekian) [inline index.js]: According to the project structure, the middleware is added to the start folder

**Comment by amirmalekian** [comment]
This feature is very interesting :)
Nice job 👍🏻

File: index.js (hunk 1 above)
Comment (yasersharifi) [reply index.js]: ok

Comment (yasersharifi) [reply index.js]: fixed changes, please merge

Comment (amirmalekian) [reply index.js]: It is only prepared for the article model, right?

Comment (yasersharifi) [reply index.js]: yes, add other model in future

**Pull request merged**